environments/docker/teardown.go

```diff
@@ -17,10 +17,9 @@ func (docker Docker) Teardown() error {
 		for _, container := range containers {
 			err = runCommand("docker rm -f " + container)
 			if err != nil {
-				fmt.Printf("Error occured while deleting docker container: %s. Error: %+v\n", container, err)
-			} else {
-				fmt.Printf("Deleted container: %s\n", container)
+				return err
```

Comment:
I think this should not return an error instead of this we should log this and continue.

Reply:
@Akash4927 This should  be handled in the test case.